netjsonconfig/schema.py

```diff
@@ -782,6 +783,9 @@
                 "country": {
                     "type": "string",
                     "maxLength": 2,
+                    "default": "IT",
```

Comment:
remove this line please, there's no reason for which the default country should be Italy ;-)